freeipa-certbot.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Renew certbot IPA certificate
+Wants=freeipa-certbot.timer
+
+[Service]
+User=root
+Type=oneshot
+ExecStart=/usr/local/sbin/ipa-certbot/renew-le.sh
+SuccessExitStatus=0, 1
```

Comment:
This should be changed to `SuccessExitStatus=1`, as this setting defines successful exit states in addition to the normal `0` exit status.
See [systemd.service#SuccessExitStatus=
](https://www.freedesktop.org/software/systemd/man/systemd.service.html#SuccessExitStatus=)